policyengine_us/variables/gov/states/mt/tax/income/mt_taxable_social_security.py

```diff
@@ -0,0 +1,77 @@
+from policyengine_us.model_api import *
+
+
+class mt_taxable_social_security(Variable):
+    value_type = float
+    entity = Person
+    label = "Montana taxable social security benefits"
+    defined_for = StateCode.MT
+    definition_period = YEAR
+    reference = "https://mtrevenue.gov/wp-content/uploads/mdocs/form%202%202021.pdf#page=6"
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.mt.tax.income.social_security.amount
+        p_irs = parameters(period).gov.irs.social_security.taxability.rate
+        # Compute the amount based on the schedule in tax form.
```

Comment:
Is this borrowed from the federal rules? Mention if so

Reply:
Yes, the rates are the same.

Reply:
i mean the logic

Reply:
No, the logic comes from the [MT tax form](https://mtrevenue.gov/wp-content/uploads/mdocs/form%202%202021.pdf#page=6).  
Also, the tax form instructions are missing/incomplete. 
[2021 instructions](https://mtrevenue.gov/wp-content/uploads/mdocs/form%202%202021%20instructions.pdf#page=31)
[2022 instructions](https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2022/12/Form-2-2022-Instructions.pdf#page=32)

Reply:
I'm asking how it compares to the logic for federal taxable social security

Reply:
Their logic is completely different.